src/actions/sources.js

```diff
@@ -131,13 +131,7 @@ function loadSourceMap(generatedSource) {
         }: Source)
     );
 
-    dispatch({ type: "ADD_SOURCES", sources: originalSources });
-
-    await dispatch(loadSourceText(generatedSource));
-    originalSources.forEach(async source => {
-      await checkSelectedSource(getState(), dispatch, source);
-      checkPendingBreakpoints(getState(), dispatch, source.id);
-    });
+    dispatch(newSources(originalSources));
```

Comment:
Oh sorry, I was saying I think we should keep `await dispatch(loadSourceText(generatedSource));`

Reply:
I tested this for every edge case I could think of, including ast pending breakpoints, on load and on reload, but there is no difference in behavior by removing this await. 

Why we want to remove this:
generated sources with source maps are often huge, for example bundle.js can be over 40,000 lines. We are loading it here eagerly, which is using resources for something that may never be displayed or used in any way. 

I will make an issue for it so if I missed any possible use case we can test it, but i do not see any reason to keep it. no tests fail, and i could not find any difference while testing.

Reply:
Looks Good! @codehag Do we want create an issue with these details, and link that in TODO comment, so they don't get lost for when we get to removing that line?

Reply:
@bomsy sounds like a great idea!